src/services/authService.js

```diff
@@ -1,6 +1,9 @@
 import http from "./httpService";
 import { returnConfigItems } from "../utilities//returnEnvironmentItems"
 
+const reportItems = returnConfigItems("endPoints","apiReportUrl");
+const contactItems = returnConfigItems("endPoints","apiContactUrl");
```

Comment:
this variable name doens't make a lot of sense to me. I would expect something more like the key

```js
const apiContactUrl = ...;
```

or 

```js
const apiContactEndpoint = ...;
```

then the functions below that use these items flow easier.

```js
return http.get(apiContactEndpoint + id + "/addresses");
```

Reply:
unless contact items is not a base url?

Reply:
fixed